src/pages/ProfilePage.js

```diff
@@ -0,0 +1,237 @@
+import {View, ScrollView} from 'react-native';
+import React, {useEffect} from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import lodashGet from 'lodash/get';
+import {parsePhoneNumber} from 'awesome-phonenumber';
+import styles from '../styles/styles';
+import Text from '../components/Text';
+import ONYXKEYS from '../ONYXKEYS';
+import Avatar from '../components/Avatar';
+import HeaderWithBackButton from '../components/HeaderWithBackButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import personalDetailsPropType from './personalDetailsPropType';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import compose from '../libs/compose';
+import CommunicationsLink from '../components/CommunicationsLink';
+import Tooltip from '../components/Tooltip';
+import CONST from '../CONST';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as Expensicons from '../components/Icon/Expensicons';
+import MenuItem from '../components/MenuItem';
+import AttachmentModal from '../components/AttachmentModal';
+import PressableWithoutFocus from '../components/PressableWithoutFocus';
+import * as Report from '../libs/actions/Report';
+import OfflineWithFeedback from '../components/OfflineWithFeedback';
+import AutoUpdateTime from '../components/AutoUpdateTime';
+import * as UserUtils from '../libs/UserUtils';
+import * as PersonalDetails from '../libs/actions/PersonalDetails';
+import FullScreenLoadingIndicator from '../components/FullscreenLoadingIndicator';
+import BlockingView from '../components/BlockingViews/BlockingView';
+import * as Illustrations from '../components/Icon/Illustrations';
+import variables from '../styles/variables';
+import ROUTES from '../ROUTES';
+
+const matchType = PropTypes.shape({
+    params: PropTypes.shape({
+        /** accountID passed via route /a/:accountID */
+        accountID: PropTypes.string,
+
+        /** report ID passed */
+        reportID: PropTypes.string,
+    }),
+});
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The personal details of all users */
+    personalDetails: personalDetailsPropType,
+
+    /** Route params */
+    route: matchType.isRequired,
+
+    /** Login list for the user that is signed in */
+    loginList: PropTypes.shape({
+        /** Date login was validated, used to show info indicator status */
+        validatedDate: PropTypes.string,
+
+        /** Field-specific server side errors keyed by microtime */
+        errorFields: PropTypes.objectOf(PropTypes.objectOf(PropTypes.string)),
```

Comment:
maaaaybe we could only mention `partnerUserID` here? Since we don't care about the errors / validatedDate?